### PR TITLE
Add killed and injured count per age group stacked

### PR DESCRIPTION
**Pull request by ReutSher1**
#1160 add killed and injured count per age group stacked Widge

### CODE REVIEW

**Comment by atalyaalon** [comment]
@shaked-hayek can you review?

File: src/models/WidgetData.ts [hunk 1]
```diff
@@ -123,6 +123,7 @@ export interface IWidgetInjuredByYearData extends IWidgetDataBase {
     count: number;
   }[];
 }
+
```
Comment (shaked-hayek) [inline src/models/WidgetData.ts:126]: why?

File: src/const/cards.const.ts [hunk 2]
```diff
@@ -17,7 +17,8 @@ export const operationalCards: WidgetName[] = [
   WidgetName.injured_count_by_accident_year,
   // WidgetName.accident_count_by_driver_type,
   WidgetName.killed_and_injured_count_per_age_group,
-  WidgetName.accident_count_by_day_night
+  WidgetName.accident_count_by_day_night,
+  WidgetName.killed_and_injured_count_per_age_group_stacked
```
Comment (shaked-hayek) [inline src/const/cards.const.ts:21]: minor - but I like to add `,` in the end of the last line as well (and then when the next line wull be added in the future no need to change this one)

File: src/services/widgets.data.service.ts [hunk 3]
```diff
@@ -47,7 +47,6 @@ export const fetchWidgets = async ({
 }: IWidgetInput): Promise<ILocationData | undefined> => {
   try {
     const widgetsUrl = getWidgetUrl({ lang, newsId, yearAgo, gpsId, city, street });
-    console.log(widgetsUrl);
```
Comment (shaked-hayek) [inline src/services/widgets.data.service.ts:50]: why?

File: src/utils/barChart.utils.ts [hunk 4]
```diff
@@ -10,7 +10,7 @@ export const barsWidgetsLabels: Record<string, Array<string>> = {
     ['textView.fatal.plural', 'textView.severe.plural', 'textView.light.plural'],
   'injured_count_by_accident_year':
     ['textView.killed.plural', 'textView.severeInjured.plural', 'textView.lightInjured.plural'],
-}
+  }
```
Comment (shaked-hayek) [inline src/utils/barChart.utils.ts:13]: I suppose this changes happed automatically in your editor, but you shouldn't push them

File: src/utils/barChart.utils.ts [hunk 5]
```diff
@@ -75,6 +75,7 @@ export function createBarWidget (
   data: IWidgetMultiBarData,
   editorBarOptions: Record<number, boolean>
 ): BarDataMap[] {
+  debugger
```
Comment (shaked-hayek) [inline src/utils/barChart.utils.ts:78]: why?

**Review by shaked-hayek — CHANGES_REQUESTED** [review]
Good, please fix my comments

File: src/components/organisms/WidgetsTemplate.tsx [hunk 6]
```diff
@@ -21,6 +21,7 @@ const WidgetsTemplate: FC = () => {
   const organizationData: OrgLogoData | undefined = logosSourceMap.find((p) => p.key === organizationName);
 
   const widgetCards = widgetsData.map((widget, index, sizeOptions) => {
+
```
Comment (shaked-hayek) [inline src/components/organisms/WidgetsTemplate.tsx:24]: Another space that shouldn't be added

File: src/models/WidgetData.ts [hunk 7]
```diff
@@ -196,4 +196,4 @@ export interface IWidgetKilledAndInjuredCountPerAgeGroup extends IWidgetDataBase
     subtitle?: string;
     title?: string;
   };
-}
+}
```
Comment (shaked-hayek) [inline src/models/WidgetData.ts:199]: Here as well

**Review by shaked-hayek — CHANGES_REQUESTED** [review]

File: src/models/WidgetData.ts (hunk 7 above)
Comment (ReutSher1) [reply src/models/WidgetData.ts:199]: It doesn't show spaces in the source code here

Comment (shaked-hayek) [reply src/models/WidgetData.ts:199]: I think maybe you added or removed a space

**Pull request merged**